Detection rules content reorg (#3693)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

:frontmatter-tags-content-type: [how-to]
:frontmatter-tags-user-goals: [manage, secure]

To create a new detection rule, follow these steps:

. Define the <<rule-types, *rule type*>>. The configuration for this step varies depending on the rule type. 
. Configure basic rule settings.
. Configure advanced rule settings (optional).
. Set the rule's schedule.
. Set up alert notifications (optional).
. Set up response actions (optional).

.Requirements
[sidebar]
--
* To create detection rules, you must have access to data views, which requires the {kibana-ref}/kibana-role-management.html[{kib} privilege] `Data View Management`.

> * You will also require permissions to enable and view detections, manage rules, manage alerts, and preview rules. These permissions depend on the user role. Refer to <<detections-permissions-section>> for more information.

Reviewer: ```suggestion
* You'll also need permissions to enable and view detections, manage rules, manage alerts, and preview rules. These permissions depend on the user role. Refer to <<detections-permissions-section>> for more information.
```
